languages/csharp/exercises/concept/constructors/.docs/after.md

```diff
@@ -0,0 +1,73 @@
+Creating an instance of a _class_ is done by calling its [_constructor_][constructors] through the [`new` operator][new]. A constructor is a special type of method whose goal is to initialize a newly created instance. [Constructors look like regular methods][constructor-syntax], but without a return type and with a name that matches the classes' name.
+
+```csharp
+class Library
+{
+    private books;
+
+    public Library()
+    {
+        this.books = 10;
+    }
+}
+
+// This will call the constructor
+var library = new Library();
+```
+
+Like regular methods, constructors can have parameters. Constructor parameters are usually stored as (private) [fields][fields] to be accessed later, or else used in some off-time calculation. Arguments can be passed to constructors just like passing arguments to regular methods.
```

Comment:
I've never heard the expression off-time before? Does it mean the same as one off?

Reply:
I meant one-off. 🤦 Fixed.